packages/core/src/Cardano/types/ProtocolParameters.ts

```diff
@@ -83,7 +84,13 @@ type NewProtocolParamsInAlonzo = {
   maxValueSize: number;
   collateralPercentage: number;
   maxCollateralInputs: number;
-  costModels: CostModels;
+  /*
+    Question: Which is the source of that data needed for the mapping? (`costModels` is temporary marked as an optional field)
+    In db-sync, in `epoch_param` table we have just `cost_model_id` col with `bigint` data type (example values are `null` or `1` in test data snapshot),
+    while in `config/network/${network}/genesis/alonzo.json` is located the property {..."costsModels": {"PlutusV1":{..}, "PlutusV2": {...}}}
+    Github thread as ref: https://github.com/input-output-hk/cardano-js-sdk/pull/416#discussion_r966731327
+  */
+  costModels?: CostModels;
```

Comment:
Please, pay attention to that one

Reply:
There's a [cost_model](https://github.com/input-output-hk/cardano-db-sync/blob/master/doc/schema.md#cost_model) table, you probably need to parse the `costs` json from it